lib/Doctrine/Persistence/Mapping/ClassMetadata.php

```diff
@@ -100,55 +78,37 @@ public function getIdentifierFieldNames();
      *
      * @return string[]
      */
-    public function getAssociationNames();
+    public function getAssociationNames() : array;
 
     /**
      * Returns a type name of this field.
      *
      * This type names can be implementation specific but should at least include the php types:
      * integer, string, boolean, float/double, datetime.
-     *
-     * @param string $fieldName
-     *
-     * @return string
      */
-    public function getTypeOfField($fieldName);
+    public function getTypeOfField(string $fieldName) : string;
 
     /**
      * Returns the target class name of the given association.
-     *
-     * @param string $assocName
-     *
-     * @return string
      */
-    public function getAssociationTargetClass($assocName);
+    public function getAssociationTargetClass(string $assocName) : string;
 
     /**
      * Checks if the association is the inverse side of a bidirectional association.
-     *
-     * @param string $assocName
-     *
-     * @return bool
      */
-    public function isAssociationInverseSide($assocName);
+    public function isAssociationInverseSide(string $assocName) : bool;
```

Comment:
Maybe refactor the parameter name to `$associationName`?
```suggestion
    public function isAssociationInverseSide(string $associationName) : bool;
```